airship.cabal

```diff
@@ -28,9 +28,9 @@ library
                    , Airship.Resource
                    , Airship.Resource.Static
                    , Airship.Route
+                   , Airship.Internal.Route
```

Comment:
We don't actually use `other-modules` ever except under very special circumstances. It certainly avoids the problem of testing internal functions. However, if this is a problem please let me know what you'd prefer.